firestore/user_reports.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+
+import dateparser
+from cachetools import cached
+from fireo import models as mdl
+
+from firestore.cachemanager import INCIDENT_CACHE, INCIDENT_STATS_CACHE, flush_cache
+
+
+class UserReport(mdl.Model):
```

Comment:
should we have a status field for user report? all user reports must be reviewed/approved/rejected by admins.

Reply:
Resolved in https://github.com/1thing-org/hatecrimetracker/pull/16